modules/cnf-performing-platform-verification-latency-tests.adoc

```diff
@@ -0,0 +1,756 @@
+// Module included in the following assemblies:
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the link:https://quay.io/repository/openshift-kni/cnf-tests?tag=latest&tab=tags[Cloud-native Network Functions (CNF) tests image] to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
+
+[IMPORTANT]
+====
+The `cnf-tests` image also includes several tests that are not supported by Red Hat at this time. Only the latency tests are supported.
+====
+
+[id="cnf-performing-end-to-end-tests-prerequisites_{context}"]
+== Prerequisites
+
+. You have installed the Performance Addon Operator.
+
+. You have configured a performance profile.
+
+. You have applied all the required CNF configurations in the cluster.
+
+. You have a pre-existing `MachineConfigPool` CR applied in the cluster. The default worker pool is `worker-cnf`. For more information about creating the cluster performance profile, see _Provisioning a worker with real-time capabilities_.
+
+[id="discovery-mode_{context}"]
+== About discovery mode
+
+Discovery mode allows you to validate the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be immediately used for another test run.
+
+Some configuration items are still created by the tests. These are specific items needed for a test to run; for example, an SR-IOV Network. These configuration items are created in custom namespaces and are cleaned up after the tests are executed.
+
+[IMPORTANT]
+====
+When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true`. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified.
```

Comment:
it is recommended to run with discovery mode, but running the tests without it will:
- fail the tests if there is no relevant PP for the worker-cnf nodes
- use the existing PP 
code reference: https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/4_latency/latency.go#L88
I believe this note was relevant for the other functional tests and not the latency ones.
@Tal-or @MarSik  please confirm/ correct me if I am wrong

Reply:
@MarSik Can you PTAL at this one please.

Reply:
This is going to customers. We want to be 100% sure they will not trash their cluster accidentally. Running without discovery mode and accidentally without focus will do that to them.

Reply:
"[IMPORTANT]
====
When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true` and `-ginkgo.focus` set to the appropriate latency test. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified by the test run.
===="